### PR TITLE
Support application hooks

### PR DESCRIPTION
**Pull request by urmastalimaa**

### CODE REVIEW

File: lib/salestation/web/responses.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Error < Dry::Struct
         attribute :context, Types::Hash.default({})
 
         def body
-          {message: message}
+          {message: message, debug_message: debug_message}
```
Comment (indrekj) [inline lib/salestation/web/responses.rb]: What do you think about including `context` as well. This is quite useful when debugging. I guess we might not want this for public endpoints though

File: lib/salestation/web/responses.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@ class Success < Dry::Struct
         constructor_type :strict
 
         attribute :status, Types::Strict::Int
-        attribute :body, Types::Strict::String
+        attribute :body, Types::Hash
```
Comment (indrekj) [inline lib/salestation/web/responses.rb]: should this be Strict::Hash?

Comment (urmastalimaa) [inline lib/salestation/web/responses.rb]: Yeah, I guess so.

File: lib/salestation/web/responses.rb (hunk 1 above)
Comment (urmastalimaa) [inline lib/salestation/web/responses.rb]: RIght, maybe leave it to the gem user to overwrite at this point.

**Comment by deiwin** [comment]
!squash

**Comment by deiwin** [comment]
I hadn't started the review yet ...

**Comment by indrekj** [comment]
Ah sorry, I saw green mark and squash... That was my green mark.

**Pull request merged**